core/core-db-repositories/src/main/resources/db/migration/mysql/V1_9__Push_notification_template_support.sql

```diff
@@ -0,0 +1,5 @@
+alter table notification_push
+    add column template_name varchar(255) default null;
+
+alter table notification_push
+    add column locale char(2) default null;
```

Comment:
char(2) vs varhcar(5)

Reply:
I've decide to use varhcar(5), why, because when storing Locale, like SIMPLIFIED_CHINESE ("zh", "CN"), in DB it will be saved like zh_CN, in this case char(2) will be short, for Locale ENGLISH, in database will be saved 'en   ', which is wrong, soo, after changing to varchar(5) it will save Locale.ENGLISH like 'en' without spaces.